documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_3_Securing_the_Platform.md

```diff
@@ -0,0 +1,59 @@
+
+## Deployment Scenarios
+In general, there are three different possibilities for the combinations of platform and connector service instances:
+* A 1:1 mapping with a connector being a single device consisting of one platform instance and one connector service instance
+* A 1:n mapping with multiple connectors on one physical device, i.e., a system with one platform instance and multiple connector service instances
+* A n:m mapping with multiple devices offering a distributed deployment for multiple connector service instances (e.g. with Kubernetes)
+
+The figure below shows the three options with their respective trusted computing bases:
+![Deployment Scenarios](./media/deployment_scenarios.png)
+#### _Fig. 4.1.3.1: Deployment Scenarios_
+
+The TCB consists of the following components:
+* All **hardware** components with access to unencrypted app data.
+* **Firmware / bootloader / UEFI** representing all software components used to bootstrap the system and initialize the hardware before starting the kernel.
+* Optionally a **hypervisor** which may be used to better isolate multiple connectors on a device by providing a Virtual Machine (VM) for each of them. The impact on the TCB is illustrated for the 1:n mapping in a comparison to a solution with OS-level virtualization, i.e., containers.
+* The **kernel** connects user space software to the hardware of the device.
+* A (container) **runtime** responsible for starting the execution of applications on the system.
+* The **connector core services** taking care of essential connector functionalities as explained in the [System Layer](../../3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_2_IDS_Connector.md).
+* An arbitrary number of **apps** as introduced in the [System Layer](../../3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_3_App_Store_and_Data_Apps.md).
+
+Additionally, the TCB may include **external components** in the surrounding infrastructure of a connector which are used for security-relevant tasks, e.g. an authorization server.
+
+It is important to note that from a component security perspective there is no specific category for the usage of cloud solutions in comparison to an on-premise deployment. In both cases, the TCB includes all components that are required to ensure the confidentiality and integrity of the transmitted and processed data and needs to fulfill all requirements defined in the certification criteria catalog. The only differences lie in the responsibility for the different layers of the software stack and ensuring their conformity.
```

Comment:
category is unclear to me in this context.

Reply:
```suggestion
It is important to note that from a component security perspective there is no need to address the usage of cloud solutions differently than an on-premise deployment. In both cases, the TCB includes all components that are required to ensure the confidentiality and integrity of the transmitted and processed data and needs to fulfill all requirements defined in the certification criteria catalog. The only differences lie in the responsibility for the different layers of the software stack and ensuring their conformity.
```